packages/vscode-extension/src/utilities/extensionContext.ts

```diff
@@ -15,15 +19,187 @@ export const extensionContext = new Proxy<ExtensionContext>({} as ExtensionConte
   },
 });
 
-let _appRootFolder: string | null = null;
+export class AppRootFolder {
```

Comment:
I think it'd be cleaner if we get rid of this class and just rely on project controlling changes to appRootFolder value